app/controllers/members/pets_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Members
+  class PetsController < ApplicationController
+    def index
+      member = User.find(params[:member_id])
+      @pets = Pet.preload(:pet_images, :pet_areas, :areas, :user, :favorites, :rooms)
+               .where(user_id:  member.id)
+               .page(params[:page]).per(20)
```

Comment:
↓にすると表示するのは member の pets であることがわかりやすくなります。

```suggestion
      @pets = member.pets.preload(:pet_images, :pet_areas, :areas, :user, :favorites, :rooms)
               .page(params[:page]).per(20)
```

Reply:
アドバイスありがとうございます。なんとなくコツというかやり方が理解できてきた気がします。